.github/workflows/cve-scanning-gradle.yml

```diff
@@ -0,0 +1,23 @@
+name: CVE Scanning for Gradle
+
+on:
+  pull_request:
+    paths:
+      - '**/build.gradle'
+      - 'allow-list.xml'
+      - '.github/workflows/cve-scanning-gradle.yml'
```

Comment:
The same question here, do we want to run this only when those files have changed?

Reply:
yes, because this is going to scan the dependencies only, and the dependencies are changed only when these files are changed. I'll add another filer to apply the scan on PRs to master branch only.